lib/irb/statement.rb

```diff
@@ -60,7 +60,8 @@ def suppresses_echo?
       end
 
       def should_be_handled_by_debugger?
-        IRB::ExtendCommand::DebugCommand > @command_class
+        require_relative 'cmd/help'
```

Comment:
I think we need `require_relative 'cmd/debug'` here too.
This method is used as
```ruby
if @context.with_debugger && statement.should_be_handled_by_debugger?
```
and it depends on calling order. The code below does not work now.
```ruby
if statement.should_be_handled_by_debugger? && @context.with_debugger
```